src/components/Profile/styles.scss

```diff
@@ -3,20 +3,22 @@
 .profile-container {
   overflow: auto;
   margin-bottom: 100px;
+  display: flex;
+  flex-direction: column;
 }
 
 .profile-header-container {
   display: flex;
   flex-flow: row nowrap;
   align-items: center;
-  justify-content: space-between;
+  justify-content: flex-start;
   background-color: $background-gray;
   width: 100%;
-  padding: 1rem;
+  padding: 0;
 }
```

Comment:
If the start conversation button is moved from header, then I think the header items could have flex-grow: 1 or some other flex styling, so that they would take the available place and not align only to the left leaving empty space on the right.

Reply:
Hmm, in layouts they are to the left but I agree, it looks a bit silly with all the empty space. I changed it now to what you suggested.